docs/02-app/01-building-your-application/01-routing/index.mdx

```diff
@@ -146,30 +146,6 @@ This is because while folders define routes, only the contents returned by `page
 
 Learn more about [Project Organization and Colocation](/docs/app/building-your-application/routing/colocation).
 
-## Server-Centric Routing with Client-side Navigation
-
-Unlike the `pages` directory which uses client-side routing, the App Router uses **server-centric routing** to align with [Server Components](/docs/getting-started/react-essentials#server-components) and [data fetching on the server](/docs/app/building-your-application/data-fetching/fetching). With server-centric routing, the client does not have to download a route map and the same request for Server Components can be used to look up routes. This optimization is useful for all applications, but has a larger impact on applications with many routes.
-
-Although routing is server-centric, the router uses **client-side navigation** with the [Link Component](/docs/app/building-your-application/routing/linking-and-navigating#link-component) - resembling the behavior of a Single-Page Application. This means when a user navigates to a new route, the browser will not reload the page. Instead, the URL will be updated and Next.js will [only render the segments that change](#partial-rendering).
-
-Additionally, as users navigate around the app, the router will store the result of the React Server Component payload in an **in-memory client-side cache**. The cache is split by route segments which allows invalidation at any level and ensures consistency across [React's concurrent renders](https://react.dev/blog/2022/03/29/react-v18#what-is-concurrent-react). This means that for certain cases, the cache of a previously fetched segment can be re-used, further improving performance.
-
-Learn more about [Linking and Navigating](/docs/app/building-your-application/routing/linking-and-navigating).
-
-## Partial Rendering
```

Comment:
Is it intentional that partial rendering is not specifically mentioned? It's an important concept to learn about.

Reply:
It's mentioned, just reshuffled a bit.